### ISSUE TITLE
Could not load `xy.hdr` errors

### ISSUE THREAD

**Issue by hood** [pmndrs/drei#1528]
Latest React/drei versions, getting `Could not load dikhololo/dikhololo_night_1k.hdr` errors breaking my app at runtime. Despite me never mentioning/consuming such asset explicitly in my code, I'm noticing outgoing (failing) HTTP calls to retrieve an asset at `https://market-assets.fra1.cdn.digitaloceanspaces.com/market-assets/hdris/dikhololo/dikhololo_night_1k.hdr`.

**Comment by tarasyarema** [comment]
Happening to me too.

**Comment by tarasyarema** [comment]
Yes, thank you very much for the response. I indeed updated the package and it resolved the issue :)

**Comment by hood** [comment]
I updated to `9.77.0` a few hours earlier, but it doesn't seem to go away.

**Comment by drcmda** [comment]
> I updated to `9.77.0` a few hours earlier, but it doesn't seem to go away.

what's the url it wants to fetch the hdr from? the old CDN broke down, we moved to another and a drei update must fix it. if you still see the same url it didn't apply the update.

but CDNs are fragile, we are thinking of base64 inlining these assets. using https://github.com/pmndrs/assets

**Comment by hood** [comment]
> > I updated to `9.77.0` a few hours earlier, but it doesn't seem to go away.
> 
> what's the url it wants to fetch the hdr from? the old CDN broke down, we moved to another and a drei update must fix it. if you still see the same url it didn't apply the update.
> 
> but CDNs are fragile, we are thinking of base64 inlining these assets. using https://github.com/pmndrs/assets

The URL for the (failing) asset request is `https://market-assets.fra1.cdn.digitaloceanspaces.com/market-assets/hdris/dikhololo/dikhololo_night_1k.hdr`,
 the error I’m getting is:
 `[Error] Failed to load resource: An SSL error has occurred and a secure connection to the server cannot be made. (dikhololo_night_1k.hdr, line 0)`
 
 Anyway is there any way for us to avoid automatically consuming those assets? In my case I’m trying to display a really low poly, small sized 3d disc-shaped object, so I don’t think I need a ~20mb hdri map at all for this use-case.

**Comment by drcmda** [comment]
that's the old url, you are not on drei latest.

the new one is https://raw.githack.com/pmndrs/drei-assets/456060a26bbeb8fdf79326f224b6d99b8bcce736/hdri/

**Comment by hood** [comment]
> that's the old url, you are not on drei latest.
> 
> the new one is https://raw.githack.com/pmndrs/drei-assets/456060a26bbeb8fdf79326f224b6d99b8bcce736/hdri/
> 
> > object, so I don’t think I need a ~20mb hdri map at a
> 
> they are 1-2mb, the pmndrs/assets hrdi's are 100-200kb. but you can make your own env
> 
> ```js
> <Environment>
>   <Lightformer position={[x, y, z]} form="ring" />
> </Environment>
> ```

Using Environment > Lightformer the way you suggested causes me type errors on `Lightformer`:

```md
Type '{ position: number[]; form: string; }' is missing the following properties from type 'Pick<Omit<ExtendedColors<Overwrite<THREE.Mesh<THREE.BufferGeometry, any>, NodeProps<THREE.Mesh<THREE.BufferGeometry, any>, any>>>, NonFunctionKeys<{ position?: any; up?: any; scale?: any; rotation?: any; matrix?: any; quaternion?: any; layers?: any; dispose?: () => void; }>> & { ...; } & EventHandlers & { ...; }, "a...': material, geometry, castShadow, receiveShadow, and 70 more.ts(2740)
```

**Comment by drcmda** [comment]
might just be a typescript issue, the thing in itself works
https://codesandbox.io/s/lwo219

**Comment by lttkachenko** [comment]
The same issue, no CDN, files a fetched from my local dev project's public folder (Next.JS project).
The error is super-strange: No error message, just stack from webpack.

**Comment by therohanmehta** [comment]
I figure out the issue that most of the time its not loading in the first render so what can you do is 
try to render the file where you can see the environment is working and then go to the Dev tools network tab

and look for the hdr, download it locally, put it into the public folder and use the below code 

import { RGBELoader } from "three/examples/jsm/loaders/RGBELoader.js";
import { useLoader, useThree } from "@react-three/fiber";

function EnviromentSetup() {
    const { scene } = useThree();
    const hdrTexture = useLoader(RGBELoader, "location of hdr");

    useEffect(() => {
      hdrTexture.mapping = THREE.EquirectangularReflectionMapping;
      scene.environment = hdrTexture;
      scene.background = hdrTexture; // Optional
    }, [hdrTexture, scene]);

    return null;
  }


<EnviromentSetup/>

this is surely work
